src/Couchbase.Lite.Support.WinUI/win.targets

```diff
@@ -0,0 +1,19 @@
+﻿<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="InjectReference_11513e9c-85e5-4aa3-93a0-c55433b2b66d" BeforeTargets="ResolveAssemblyReferences">
+        <ItemGroup>
+            <Content Condition=" '$(Platform)' == 'x64' " Include="$(MSBuildThisFileDirectory)..\runtimes\win10-x64\native\LiteCore.dll">
+                <Link>LiteCore.dll</Link>
+                <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            </Content>
+		<Content Condition=" '$(Platform)' == 'arm64' " Include="$(MSBuildThisFileDirectory)..\runtimes\win10-arm64\native\LiteCore.dll">
+                <Link>LiteCore.dll</Link>
+                <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            </Content>
+        </ItemGroup>
+        <ItemGroup>
+            <SDKReference Include="Microsoft.VCLibs, Version=14.3">
+              <Name>Visual Studio 2022 C++ Runtime for Universal Windows Platform Apps</Name>
+            </SDKReference>
+        </ItemGroup>
```

Comment:
Remove this ItemGroup, it will actually cause problems with using the package as I have found.